modules/end-user-guide/pages/authenticating-yourself-to-a-git-server-from-a-workspace.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: CONCEPT
+:description: Authenticating yourself to a Git server from a workspace
+:keywords: authentication, authenticate, github, gitlab, bitbucket
+:navtitle: Authenticating yourself to a Git server from a workspace
+// :page-aliases:
+
+[id="authenticating-yourself-to-a-git-server-from-a-workspace_{context}"]
+= Authenticating yourself to a Git server from a workspace
+
+In a workspace, you can run Git commands that require user authentication like cloning a remote private Git repository or pushing to a remote public or private Git repository.
+
+To configure user authentication to a Git server from a workspace, {prod-short} provides two options:
+
+* Your administrator sets up an xref:administration-guide:oauth-for-github-gitlab-or-bitbucket.adoc[OAuth application on GitHub, GitLab, or Bitbucket] for your organization's {prod} instance.
+
+* You create your own, user xref:creating-a-kubernetes-secret-for-a-git-credentials-store.adoc[{kubernetes} Secret for a Git credentials store].
+
+.Additional resources
+* xref:administration-guide:oauth-for-github-gitlab-or-bitbucket.adoc[Administration Guide: OAuth for GitHub, GitLab, or Bitbucket]
+* xref:creating-a-kubernetes-secret-for-a-git-credentials-store.adoc[User Guide: Creating a Kubernetes Secret for a Git credentials store]
```

Comment:
ERROR (asciidoctor): target of xref not found: creating-a-kubernetes-secret-for-a-git-credentials-store.adoc